charts/flyte-core/values.yaml

```diff
@@ -424,6 +424,8 @@ common:
     # -- Specify your Secret (with sensitive data) or pseudo-manifest (without sensitive data). See https://github.com/godaddy/kubernetes-external-secrets
     secretManifest: {}
   ingress:
+    # --- Sets the ingressClassName
+    class:
```

Comment:
I'd prefer to keep this key name as `ingressClassName`. `flyte-binary` uses that structure too